doc/README.md

```diff
@@ -22,7 +22,7 @@ Unpack the files into a directory and run:
 
 ### Windows
 
-Unpack the files into a directory, and then run 
```

Comment:
trailing whitespace generated error in `test/lint/lint-whitespace.sh`
https://travis-ci.org/bitcoin/bitcoin/jobs/425139871